os-eap70-openshift/added/standalone-openshift.xml

```diff
@@ -464,19 +464,25 @@
         <interface name="private">
             <inet-address value="${jboss.bind.address.private:127.0.0.1}"/>
         </interface>
+        <interface name="bindall">
```

Comment:
I don't see why we need to define a new interface, can't we just set the  jboss.bind.address  (via -b 0.0.0.0 )?

Reply:
I kept this way just because eap6 is configured in the same way.
I am comfortable with both options. @rcernich wdty?